crates/sui-indexer-alt-graphql/README.md

```diff
@@ -0,0 +1,110 @@
+# sui-indexer-alt-graphql
+
+A GraphQL service that answers queries based on indexed data.
+
+## Set-up
+
+GraphQL requires access to one or more Data Sources (see below), so these need
+to be set-up and accessible for GraphQL to run.
+
```

Comment:
Does it make sense to add a section for dependencies? 
I had to install `nextest` and `insta` for testing, should they be listed somewhere in a `Cargo.toml` or is this a global dependency that we have to install ourselves (I am Rust noob so not familiar with package management best practices).

```
brew install nextest # test runner
curl -LsSf https://insta.rs/install.sh | sh # snapshot testing
```